packages/cadl-ranch-specs/cadl-ranch-summary.md

```diff
@@ -670,6 +670,99 @@ Expected response body:
 }
 ```
 
+### Client_Naming_header
+
+- Endpoint: `post /client/naming/header`
+
+Testing that we can project a header name.
+Your generated SDK should generate an operation header `parameter` with a single parameter called `clientName`.
+
+Expected header parameter: `default-name="true"`
```

Comment:
small nit: would this be better split into `client/naming/header/request` and `client/naming/header/response`